### DISCUSSION TITLE
How to connect to SFTPGo server from macos terminal

### DISCUSSION THREAD

**Discussion by umeshauv** [drakkan/sftpgo#1149] [General]
Hi
  I have run the SFTPGo server as a docker container as specified in https://github.com/drakkan/sftpgo/blob/main/docker/README.md 
  How can I use sftp command line from mac os terminal to connect to the SFTP server by executing the command 
```
sftp -P 2022 admin@127.0.0.1 or sftp -P 2022 admin@docker.host.internal
```
I am able to access webadmin, but not able to connect to SFTP server.  

Thanks for your help.

Umesh

**Comment by JonBendtsen** [comment]
What error message are you getting when you try to access using SFTP?

And can we get output from this command `docker ps | grep -i sftpgo`

Or even better from this command
```
docker inspect some-sftpgo | jq '.[0].NetworkSettings.Ports'
```
